assembl/static2/js/app/components/debate/survey/question.jsx

```diff
@@ -55,21 +69,36 @@ class Question extends React.Component {
           });
         })
     );
-  }
+  };
+
+  updateBody = (newValue) => {
+    this.setState({
+      postBody: newValue
+    });
+  };
 
-  redirectToLogin() {
+  redirectToLogin = () => {
     const isUserConnected = getConnectedUserId();
     const { scrollToQuestion, index } = this.props;
     if (!isUserConnected) {
       inviteUserToLogin();
     } else {
       scrollToQuestion(true, index);
     }
-  }
+  };
+
+  updateCharCount = (newValue) => {
+    this.setState({ charCount: newValue });
+  };
 
   render() {
     const { index, title, screenWidth, screenHeight } = this.props;
-    const height = screenHeight - document.getElementById('timeline').clientHeight;
+    let height = 0;
+    const timeline = document && document.getElementById('timeline');
+    // This is necessary to bypass an issue with Flow
```

Comment:
https://stackoverflow.com/questions/44979394/why-flow-still-complains-about-null-values-for-document-getelementbyid